apps/vaporgui/PGroup.h

```diff
@@ -19,5 +18,20 @@ class PGroup : public PWidget {
     PGroup *Add(PWidget *pw);
 
 protected:
+    QWidget *_widget;
```

Comment:
Currently, all other PWidgets hold their QWidgets/VWidgets as private members.  Is there a reason that it is protected here, and private everywhere else?  Making them protected amongst all PWidgets would be more consistent, and allow all specialized PWidgets to be inheritable.

Reply:
I made it private.